optional_requirements.txt

```diff
@@ -1,3 +1,7 @@
-pynufft==2020.2.7
 numba
+llvmlite
```

Comment:
A compiler?

Reply:
Should this not be installed by numba implicitly if numba requires it?

Reply:
Yeah will remove it.